src/dwn-server.ts

```diff
@@ -41,6 +41,11 @@ export class DwnServer {
     callback?.();
   }
 
+  /**
+   * Handler for creating a DWN if none exists.
```

Comment:
this seems like a confusing way to describe this function. I'm not sure "handler" is the correct term (maybe we could just call it a function?) and the primary purpose of this function is to setup the servers (http and websocket), the DWN creation is secondary and only happens if it hasn't already been done.